api/server/routes_create.go

```diff
@@ -89,9 +86,9 @@ func (s *Server) handleRouteCreate(c *gin.Context) {
 
 	}
 
-	route, err := s.Datastore.InsertRoute(ctx, wroute.Route)
+	route, err := s.Datastore.InsertRoute(c, wroute.Route)
 	if err != nil {
-		handleErrorResponse(c, err)
+		handleErrorResponse(c, r, err)
```

Comment:
Yeah, we can use just *gin.Context, but is it that bad to use it? And I think it provides clearly what kind of information an API request handler can have.
Should we remove it?